mito-ai/mito_ai/prompt_builders/chat_system_message.py

```diff
@@ -37,6 +44,9 @@ def create_chat_system_message_prompt() -> str:
 
 Notice in the example above that the citation uses line number 2 because citation line numbers are 0-indexed.
 
+===
+{get_database_rules()}
```

Comment:
Putting this in the system prompt so we don't send it over and over again makes sense. 

Eventually, it might even make sense to move this to a tool use that the agent and chat are allowed to use called `check_snowflake_schema` or something like that so it can decide when to use it...